synedrion/src/uint/traits.rs

```diff
@@ -239,10 +238,11 @@ impl HasWide for U4096 {
     }
 }
 
-pub type U512Mod = MontyForm<{ nlimbs!(512) }>;
-pub type U1024Mod = MontyForm<{ nlimbs!(1024) }>;
-pub type U2048Mod = MontyForm<{ nlimbs!(2048) }>;
-pub type U4096Mod = MontyForm<{ nlimbs!(4096) }>;
+// TODO(dp): Suggest crypto-bigint update nlimbs! macro.
+pub type U512Mod = MontyForm<{ 512u32.div_ceil(Limb::BITS) as usize }>;
```

Comment:
This is not necessary, I pinned Rust version for the clippy CI action, it now runs on 1.81.

Reply:
I just run it with `+1.81`